web/index.html

```diff
@@ -266,13 +266,20 @@ <h1>Mand - Redirect URL's</h1>
      // Add each link to the dom, can be done better by first building link elements and then adding
      links.forEach((l) => {
        let link = document.createElement('div');
-       link.innerHTML = `
-         <span>${l.Name}</span>
-         <div>
-           <img src="/admin/to.svg"/>
-         </div>
-         <a href="${l.Redirect}">${l.Redirect}</a>
-       `;
+
+       let name = document.createElement('span');
+       let href = document.createElement('a');
+       let to = document.createElement('div');
+       let img = document.createElement('img');
+       name.textContent = l.Name;
+       href.textContent = l.Redirect;
+       href.href = l.Redirect;
```

Comment:
This is the only one left to XSS. You can probably do `new URL(l.Redirect).href` instead. That does not a polyfill for IE11 though, if you care about that.

Reply:
Apologies, it requires more handling. You need to check if the protocol is `http:` or `https:` as well.

Reply:
Are you satisfied with the current solution? :)